content/08-cardano-sidechains/03-example-evm-sidechain/06-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+
+- Get transaction index for the epoch the lock transaction will be processed by invoking the `sidechain_getOutgoingTransactions` API method:
+
+```
+curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "sidechain_getOutgoingTransactions",
+  "params": [<N or N+1>],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Example output:
+
+```json
+{
+  "proof": {
+    "bytes": "0xd8799fd8799f0002581d606e9d4f6a3f900f7b510b27f29d8e79c550686ce093946b23b3d1828ed87a80ff80ff",
+    "info": {
+      "transaction": {
+        "value": "0x2",
+        "recipient": "0x606e9d4f6a3f900f7b510b27f29d8e79c550686ce093946b23b3d1828e",
+        "txIndex": 0
+      },
+      "merkleRootHash": "0x96e0bffb3786607057ddffa5bc3a92c5b6669109c12b5222da374cd99a815b4e"
+    }
+  }
+}
+```
+
+To obtain the `merkleProof` matching the recipient and amount used to lock tokens, proceed as follows.
+
+- Wait until the epochPhase changes to `handover` of epoch `N` or `N+1` and obtain the Merkle proof for your lock action by invoking the `sidechain_getOutgoingTxMerkleProof` API method:
+
+```shell
+curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "sidechain_getOutgoingTxMerkleProof",
+  "params": [<N or N+1>, <txIndex>],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Example: 
+
+```
+curl -L -X POST https://faucet.sidechain.evmtestnet.iohkdev.io/ -H 'Content-type:application/json' -d '{"jsonrpc": "2.0", "method": "sidechain_getOutgoingTxMerkleProof", "params": [1295, 1], "id": 1}' | jq
+
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "proof": {
+      "bytes": "0xd8799fd8799f0101581d6045736b33c53d9f526a223086afd812b07ed9741da80e989e96998893d8799f5820772485d60f6744cf252f26560413aae8d28c82a88b1c77eede792f28965f4e79ffff9fd8799f005820ed69142610619b748ec5cd657e418c1c891c3a176900376d12db0b3c406a0a38ffffff",
+      "info": {
+        "transaction": {
+          "value": "0x1",
+          "recipient": "0x6045736b33c53d9f526a223086afd812b07ed9741da80e989e96998893",
+          "txIndex": 1
+        },
+        "merkleRootHash": "0xaacd27079b9c8b8bd17693beccb8997b4b7b1ef97596018d43047a40dab6b021"
+      }
+    },
+    "sidechainEpoch": 1295
+  },
+  "id": 1
+}
+```
+
+From the return you need to obtain the `proof/bytes` and remove the `0x` that is prepended. Call that string `merkleProof`. 
+
+*In the above example `d8799fd8799...` is the Merkle proof.*
+
+- Wait until the epoch changes and the committee handover has happened. The committee handover happens automatically on the testnets by a service (relay) that tries every minute to find epoch signatures, and succeeds after a new epoch has started and the signatures are obtained. On mainnet, the committee handover will be done by anyone who wants, where there will be an incentive to do so. A good way to verify this is to wait until you observe your transaction in the list of signatures to upload:
```

Comment:
Perhaps add "A sidechain epoch is 1 hour. 30 first minutes is the regular phase, 15 minutes closedTxBatch and 15 minutes handover phase"

Reply:
I added a note describing the your suggestion.